tests/Unit/Application/Commands/DeleteShoppingItem/DeleteShoppingItemCommandTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Tests\Unit\Application\Commands\DeleteShoppingItem;
+
+use Lindyhopchris\ShoppingList\Application\Commands\DeleteShoppingItem\DeleteShoppingItemModel;
+use Lindyhopchris\ShoppingList\Domain\ShoppingItem;
+use Lindyhopchris\ShoppingList\Domain\ShoppingItemStack;
+use Lindyhopchris\ShoppingList\Domain\ShoppingList;
+use Lindyhopchris\ShoppingList\Persistance\ShoppingListRepositoryInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class DeleteShoppingItemCommandTest extends TestCase
+{
+    /**
+     * @var ShoppingListRepositoryInterface|MockObject
+     */
+    private ShoppingListRepositoryInterface|MockObject $repository;
+
+    /**
+     * @var DeleteShoppingItemCommandTest
+     */
+    private DeleteShoppingItemCommandTest $command;
+
+    /**
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->command = new DeleteShoppingItemCommandTest(
```

Comment:
Comment: you've used the wrong class here (it's likely a typo when using PHPStorm predictive text). It should be `DeleteShoppingItemCommand` not `DeleteShoppingItemCommandTest`.

Reply:
Edited this after I pushed the code! I think it was a simple mistakes for sure.